src/Uno.UI/UI/Xaml/Controls/Frame/Frame.cs

```diff
@@ -39,6 +39,17 @@ public Frame()
 
 		internal PageStackEntry CurrentEntry { get; set; }
 
+		protected override void OnContentChanged(object oldValue, object newValue)
+		{
+			base.OnContentChanged(oldValue, newValue);
+
+			// Make sure we void CurrentEntry when someone sets Frame.Content = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5436574)